src/jukebox/components/rfid/hardware/generic_nfcpy/generic_nfcpy.py

```diff
@@ -0,0 +1,130 @@
+# Standard imports from python packages
+import logging
+
+import nfc
+import glob
+from nfc.clf import RemoteTarget
+
+# Import the ReaderBaseClass for common API. Leave as this line as it is!
+from components.rfid import ReaderBaseClass
+import jukebox.cfghandler
+import misc.inputminus as pyil
+from misc.simplecolors import Colors
+
+# Also import the description string into this module, to make everything available in a single module w/o code duplication
+# Leave this line as is!
+from .description import DESCRIPTION
+
+# Create logger.
+logger = logging.getLogger('jb.rfid.nfcpy')
+# Get the global handler to the RFID config
+cfg = jukebox.cfghandler.get_handler('rfid')
+
+
+def query_customization() -> dict:
+    possible_device_ids = [
```

Comment:
Does this need to be specified again (already in setup.inc.sh)?
As far as i understand the setup.inc.sh creates the `/etc/udev/rules.d/50-usb-nfc-rule.rules` file, so only this devices can really be used. Maybe parse this file here and only use the available entries?

Reply:
I also don't like there is no single point of truth, but I also don't like the idea of parsing an udev file. nfcpy itself must have a list of valid VID/PID pairs. I dig into it to see what I can do...

Reply:
Ok 👍
But if it's not trivial this is no stopper. We can also change it later if you find something